modules/moordyn/src/MoorDyn_Registry.txt

```diff
@@ -24,7 +24,8 @@ typedef   MoorDyn/MD  InitInputType    ReKi               g              -   -99
 typedef   ^           ^                ReKi               rhoW           -   -999.9 -   "sea density"        "[kg/m^3]"
 typedef   ^           ^                ReKi               WtrDepth       -   -999.9 -   "depth of water"     "[m]"
 typedef   ^           ^                ReKi               PtfmInit      {:}{:} -    -   "initial position of platform(s) shape: 6, nTurbines"     -
-typedef   ^           ^                IntKi              FarmSize       -    0     -   "Indicates normal FAST module mode if 0, FAST.Farm coupled mode and =nTurbines if >0"     -
+typedef   ^           ^                IntKi              FarmSize       -    0     -   "Indicates normal FAST module mode if 0, FAST.Farm coupled mode and =nTurbines if >0, standalone mode if -1"     -
+typedef   ^           ^                IntKi              Standalone     -    0     -   "Indicates MoorDyn run as standalone code if 1, coupled if 0"     -
```

Comment:
Seems like the value of `FarmSize` is sufficient to determine `Standalone`, so this seems redundant.

Reply:
Agreed, good catch. I am removing them in the next commit